libkbfs/semaphore_disk_limiter.go

```diff
@@ -52,24 +53,49 @@ func (sdl semaphoreDiskLimiter) beforeBlockPut(
 	availableBytes, availableFiles int64, err error) {
 	if blockBytes == 0 {
 		// Better to return an error than to panic in Acquire.
-		return sdl.s.Count(), defaultAvailableFiles, errors.New(
+		return sdl.bytesSemaphore.Count(), defaultAvailableFiles, errors.New(
 			"semaphore.DiskLimiter.beforeBlockPut called with 0 blockBytes")
 	}
 
-	availableBytes, err = sdl.s.Acquire(ctx, blockBytes)
+	availableBytes, err = sdl.bytesSemaphore.Acquire(ctx, blockBytes)
 	return availableBytes, defaultAvailableFiles, err
 }
 
 func (sdl semaphoreDiskLimiter) afterBlockPut(
 	ctx context.Context, blockBytes, blockFiles int64, putData bool) {
 	if !putData {
-		sdl.s.Release(blockBytes)
+		sdl.bytesSemaphore.Release(blockBytes)
 	}
 }
 
 func (sdl semaphoreDiskLimiter) onBlockDelete(
 	ctx context.Context, blockBytes, blockFiles int64) {
 	if blockBytes > 0 {
-		sdl.s.Release(blockBytes)
+		sdl.bytesSemaphore.Release(blockBytes)
+	}
+}
+
+type semaphoreDiskLimiterStatus struct {
+	Type string
+
+	MaxJournalMB         float64
+	MBSemaphoreCount     float64
```

Comment:
Same here, seems inconsistent with above line.  Maybe it's on purpose?  But is confusing to me.